src/site/stages/build/process-cms-exports/schemas/raw/paragraph-link_teaser.js

```diff
@@ -0,0 +1,25 @@
+const { getFilter } = require('../../filters');
+
+/* eslint-disable camelcase */
+module.exports = {
+  type: 'object',
+  properties: {
+    field_link: {
+      type: 'array',
+      maxItems: 1,
```

Comment:
How confident are you that this will always be only one item long?

Reply:
_Reasonably_ confident?  It seemed that a `link_teaser` was a singular thing based on how the transformed object looked:

```json
  "entity": {
    "fieldLink": {
      "url": {
        "path": "https://www.benefits.va.gov/gibill/my_story.asp"
      },
      "title": "My Story—How Veterans and Servicemembers Use the GI Bill",
      "options": []
    },
    "fieldLinkSummary": "Hear inspiring stories from Veterans and Servicemembers who’ve advanced their education and training with the Post-9/11 GI Bill."
  }
```